telemetry/definitions/commonDefinitions.json

```
@@ -1413,6 +1413,18 @@
             "type": "string",
             "description": "Language used for the project."
         },
+        {
+            "name": "languageServerResourceLocation",
```

Why do we continually type out "language server" everywhere? It is less consistent and harder to guess (when grepping) than just typing "lsp". It's also potentially "wrong" some of these metrics will be related to the "client", not the "server". 

```suggestion
            "name": "lspResourceLocation",
```

---

I agree, but it does look like both eclipse and VS use the `languageServer` prefix, so we'd have to come to agreement on this transition.